app/src/main/java/org/oppia/app/topic/practice/TopicPracticeFragmentPresenter.kt

```diff
@@ -168,13 +167,4 @@ class TopicPracticeFragmentPresenter @Inject constructor(
       topicPracticeFooterViewBinding.isSubmitButtonActive = skillIdHashMap.isNotEmpty()
     }
   }
```

Comment:
Not able to comment on that particular line but writing here. 
There is an unused import at line number 14 `import org.oppia.app.model.EventLog`

Reply:
Done.